diff --git a/stm32c0xx/Include/stm32c011xx.h b/stm32c0xx/Include/stm32c011xx.h
index 5677848..cf8d0a0 100644
--- a/stm32c0xx/Include/stm32c011xx.h
+++ b/stm32c0xx/Include/stm32c011xx.h
@@ -610,7 +610,6 @@ typedef struct
 
 #define DMAMUX1_ChannelStatus      ((DMAMUX_ChannelStatus_TypeDef *) DMAMUX1_ChannelStatus_BASE)
 #define DMAMUX1_RequestGenStatus   ((DMAMUX_RequestGenStatus_TypeDef *) DMAMUX1_RequestGenStatus_BASE)
-#define DMAMUX1_IdRegisters        ((DMAMUX_IdRegisters_TypeDef *) DMAMUX1_IdRegisters_BASE)
 
 #define DBG              ((DBG_TypeDef *) DBG_BASE)
 
diff --git a/stm32c0xx/Include/stm32c031xx.h b/stm32c0xx/Include/stm32c031xx.h
index 65a6e91..ad8f970 100644
--- a/stm32c0xx/Include/stm32c031xx.h
+++ b/stm32c0xx/Include/stm32c031xx.h
@@ -614,7 +614,6 @@ typedef struct
 
 #define DMAMUX1_ChannelStatus      ((DMAMUX_ChannelStatus_TypeDef *) DMAMUX1_ChannelStatus_BASE)
 #define DMAMUX1_RequestGenStatus   ((DMAMUX_RequestGenStatus_TypeDef *) DMAMUX1_RequestGenStatus_BASE)
-#define DMAMUX1_IdRegisters        ((DMAMUX_IdRegisters_TypeDef *) DMAMUX1_IdRegisters_BASE)
 
 #define DBG              ((DBG_TypeDef *) DBG_BASE)
 
